### ISSUE TITLE
Add type definitions to generated files

### ISSUE THREAD

**Issue by spawnia** [kylekatarnls/carbon-cli#1]
Thanks @kylekatarnls for this package and your great work on Carbon.

Given the following macro:

```php
Carbon::macro('foo', function(): string {});
```

I would expect the generated definition to contain a return type.
The output should be changed as follows.

```diff
namespace Carbon
{
    class Carbon
    {
+       public function formatHyphenlessDate(): string
-       public function formatHyphenlessDate()
        {
            // Content, see foo.php
        }
    }
}
```

**Comment by kylekatarnls** [comment]
Fixed in [1.2.0](https://github.com/kylekatarnls/carbon-cli/releases/tag/1.2.0)

**Comment by spawnia** [comment]
Sweet, good job.
